lib/index.js

```diff
@@ -27,6 +28,7 @@ exports.agents = {
 exports.request = function (method, url, options, callback, _trace) {
 
     options = options || {};
+    var logger = new Logger();
```

Comment:
no reason to new this up or to have a constructor for the logger.

Lets make Logger export a log function only.  If logging isn't active make it a Hoek.ignore function.